tree_test.go

```diff
@@ -352,10 +384,14 @@ func TestTreeInsertAndGet(t *testing.T) {
 			inserts: []testInsert{
 				{
 					network: "1.1.0.0/24",
+					start:   "1.1.0.0",
+					end:     "1.1.0.255",
 					value:   mmdbtype.Map{"a": mmdbtype.Slice{mmdbtype.Uint64(1), mmdbtype.Bytes{1, 2}}},
 				},
 				{
 					network: "1.1.1.0/24",
+					start:   "1.1.1.0",
+					end:     "1.1.1.255",
```

Comment:
If it isn't too difficult, it would be nice to have at least one test that inserted a range that included multiple CIDR networks.

Reply:
Added in https://github.com/maxmind/mmdbwriter/pull/25/commits/b00a7125e80388b28ca99414106e10a72d2df01c